setup.py

```diff
@@ -16,29 +16,36 @@
 #   TORCH_XLA_PACKAGE_NAME
 #     change the package name to something other than 'torch_xla'
 #
-#   COMPILE_PARALLEL=1
-#     enable parallel compile
-#
-#   BUILD_CPP_TESTS=1
-#     build the C++ tests
-#
-#   XLA_DEBUG=0
-#     build the xla/xrt client in debug mode
-#
-#   XLA_BAZEL_VERBOSE=0
+#   BAZEL_VERBOSE=0
 #     turn on verbose messages during the bazel build of the xla/xrt client
 #
 #   XLA_CUDA=0
 #     build the xla/xrt client with CUDA enabled
 #
+#   XLA_CPU_USE_ACL=0
+#     whether to use ACL
+#
 #   BUNDLE_LIBTPU=0
 #     include libtpu in final wheel
 #
+#   GCLOUD_SERVICE_KEY_FILE=''
+#     file containing the auth tokens for remote cache/build. implies remote cache.
+#
+#   BAZEL_REMOTE_CACHE=""
+#     whether to use remote cache for builds
```

Comment:
is there a guidance about on what occasion we should or should not use remote cache?

Reply:
I'll add documentation to this PR, but generally we should be using it always.

We want to minimize cache misses so maybe each separate environment can use it's own
`SILO_NAME`, but CI and cloudbuild have their custom ones set, others can use `dev`
by default and we can see how that goes.

GCloud authentication is required for this feature.